Resources/Prototypes/_LostParadise/Entities/Objects/Weapons/Guns/Ammunition/Projectile/LPPBullets.yml

```
@@ -167,3 +167,30 @@
+      types:
+        Blunt: 10
+  - type: StaminaDamageOnCollide
+    damage: 60
+
+- type: entity
+  noSpawn: true
+  parent: BaseBullet
+  id: LPPBulletAntiMaterielCP
+  name: bullet (.60 Concrete projectile)
+  components:
+  - type: Projectile
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)